xc/common/cmake/project_ray.cmake

```diff
@@ -36,7 +36,17 @@ function(PROJECT_RAY_ARCH)
   set(ARCH_IMPORT ${symbiflow-arch-defs_SOURCE_DIR}/xc/common/utils/prjxray_arch_import.py)
   set(CREATE_SYNTH_TILES ${symbiflow-arch-defs_SOURCE_DIR}/xc/common/utils/prjxray_create_synth_tiles.py)
   set(CREATE_EDGES ${symbiflow-arch-defs_SOURCE_DIR}/xc/common/utils/prjxray_create_edges.py)
-  set(DEPS ${PRJRAY_DB_DIR}/${PRJRAY_ARCH}/${PART}/tilegrid.json)
+  set(GET_FABRIC ${symbiflow-arch-defs_SOURCE_DIR}/xc/common/utils/prjxray_get_fabric.py)
+  execute_process(
+    COMMAND ${CMAKE_COMMAND} -E env PYTHONPATH=${PRJRAY_DIR}:${symbiflow-arch-defs_SOURCE_DIR}/utils
+    ${PYTHON3} ${GET_FABRIC}
+      --db_root ${PRJRAY_DB_DIR}/${PRJRAY_ARCH}/
+      --part ${PART}
+      -cmake
+    OUTPUT_VARIABLE FABRIC
+    )
+  string(REPLACE "\n" "" FABRIC "${FABRIC}")
```

Comment:
I think you don't need the `string(REPLACE` anymore, because you added the `end=""`?

Reply:
Well, the bash prints a newline at the beginning of the output. That's why I need this replace. I could remove the end="" in the print because that isn't necessary now.

Reply:
Ya, remove the `end=""` in that case.